cmd/agent/main.go

```diff
@@ -79,6 +79,9 @@ func main() {
 			if err := agent.Run(); err != nil {
 				return fmt.Errorf("failed to run the agent: %w", err)
 			}
+
+			app.ExposeTuningOptions(builder)
```

Comment:
I'm not sure I'm a big fan of having the command to call this. Sounds like a knowledge it shouldn't have. How about providing this as a private function, which is called by `Start()/Run()`? Or even closer to where they are defined/used, perhaps in the builder themselves. The badger storage plugin does something similar when it's being initialized:

https://github.com/jaegertracing/jaeger/blob/a2fb067f01551b47aef4f58bcc0e925ffeb32f5f/plugin/storage/badger/factory.go#L137